common/api/endpoint/ApiEndPoint.ts

```diff
@@ -5,6 +5,11 @@ export const ApiEndPoint = {
   Init: {
     one: () => apiPath('init'),
   },
+  CycleData: {
+    PersistNode: {
+      one: () => apiPath('cycleData', 'persistNode'),
+    },
+  },
```

Comment:
how we should define the query params used for this (e.g. countryIso, assessmentName, cycleName...)

Reply:
@minotogna

Reply:
We discussed this in our meeting a while ago

Reply:
I don't remember. :(

Reply:
I see, I misunderstood the question. 
I think the action itself defines the params. would that be enough already?